urlparse4/__init__.py

```diff
@@ -1,8 +1,11 @@
 # https://github.com/python/cpython/blob/2.7/Lib/urlparse.py
 
-from urlparse import *
+import six
+
+from six.moves.urllib.parse import urlparse, urlsplit, urljoin
+
 
 _original_urlsplit = urlsplit
 _original_urljoin = urljoin
 
-from cgurl import urlsplit, urljoin
+from cgurl import url_split, url_join
```

Comment:
this functions should override urlsplit, urljoin from urlparse, so names should stay the same

Reply:
I had some ImportError before so I tried to change it as an experiment. I will put it back now 😄